include/fluent-bit/flb_config.h

```diff
@@ -269,6 +269,11 @@ struct flb_config {
     int shutdown_by_hot_reloading;
     int hot_reloading;
 
+#ifdef FLB_SYSTEM_WINDOWS
+    /* maxstdio (Windows) */
+    int maxstdio;
```

Comment:
use a prefix as well for the internal variable: `win_maxstio`

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/9707/commits/beff608dc69d81aab78a03eae041b25438783efe.